src/console/console_cmd.c

```diff
@@ -294,6 +349,92 @@ int console_cmd_rank(game_state *gs, int argc, char **argv) {
     return 1;
 }
 
+int console_cmd_assert(game_state *gs, int argc, char **argv) {
+    if(argc != 4) {
+        console_output_addline("Usage: assert harX.attr OP value");
+        return -1;
+    }
+
+    // check we have HARs
+    game_player *player = game_state_get_player(gs, 0);
+    object *har_obj = game_state_find_object(gs, game_player_get_har_obj_id(player));
+    if(!har_obj) {
+        return 1;
+    }
+
+    // Parse LHS: harX.attr
+    char *lh_har_part = argv[1];
+    char *dot_pos = strchr(lh_har_part, '.');
+    if(dot_pos == NULL) {
+        console_output_addline("Invalid LHS format. Expected harX.attr");
+        return 1;
+    }
+
+    // Split into har and attribute parts
+    *dot_pos = '\0'; // Terminate har part
+    char *lh_attr_part = dot_pos + 1;
+
+    rec_assertion op;
+
+    int res = rec_assertion_get_operand(&op.operand1, lh_har_part, lh_attr_part);
+    if(res == 1) {
+        console_output_addline("Invalid LHS har identifier. Use har1 or har2.\n");
+        return 1;
+    } else if(res == 2) {
+        console_output_addline("Invalid LHS har attribute");
+    }
+
+    // Parse operator
+    const char *oper = argv[2];
+
+    if(strcmp(oper, ":=") == 0) {
+        op.op = OP_SET;
+    } else if(strcmp(oper, "==") == 0) {
```

Comment:
Please support "gt|lt|eq" names for operators for parity with rectool.

Reply:
Done.